http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpsClientPem.java

```diff
@@ -0,0 +1,122 @@
+package io.airlift.http.client.jetty;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.http.client.AbstractHttpClientTest;
+import io.airlift.http.client.HttpClientConfig;
+import io.airlift.http.client.Request;
+import io.airlift.http.client.ResponseHandler;
+import io.airlift.http.client.TestingRequestFilter;
+import io.airlift.http.client.spnego.KerberosConfig;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.concurrent.TimeoutException;
+
+import static com.google.common.io.Resources.getResource;
+import static io.airlift.http.client.Request.Builder.prepareGet;
+import static io.airlift.testing.Closeables.closeQuietly;
+
+public class TestJettyHttpsClientPem
+        extends AbstractHttpClientTest
+{
+    private JettyHttpClient httpClient;
+
+    TestJettyHttpsClientPem()
```

Comment:
Make public since it will be constructed by TestNG (which probably bypasses security just fine but seems bad to rely on that)